document_merge_service/settings.py

```diff
@@ -250,11 +250,13 @@ def parse_admins(admins):
 URL_PREFIX = env.str("URL_PREFIX", default="")
 
 # Email settings
+SERVER_EMAIL = env.str("SERVER_EMAIL", default="root@localhost")
```

Comment:
We should expose the `DEFAULT_FROM_EMAIL` variable as well: https://docs.djangoproject.com/en/4.1/ref/settings/#std-setting-DEFAULT_FROM_EMAIL

Reply:
Forgot about that one, thanks :)